packages/beacon-node/test/unit/network/unavailableBeaconBlobsByRoot.test.ts

```diff
@@ -172,7 +178,7 @@ describe("unavailableBeaconBlobsByRoot", () => {
     ];
 
     const blockData = {
-      fork: ForkName.deneb as ForkBlobs,
+      fork: ForkName.deneb as ForkPostDeneb,
```

Comment:
probably the right approach here would be to use ` as const`

```suggestion
      fork: ForkName.deneb as const,
```